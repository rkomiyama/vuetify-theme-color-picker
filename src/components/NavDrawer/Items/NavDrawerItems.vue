<template>
  <v-list
    dense
    expand
    nav
  >
    <ThemeListGroup />
  </v-list>
</template>

<script>
import ThemeListGroup from './ThemeListGroup'

export default {
  name: 'NavDrawerItems',
  components: {
    ThemeListGroup
  }
}
</script>
