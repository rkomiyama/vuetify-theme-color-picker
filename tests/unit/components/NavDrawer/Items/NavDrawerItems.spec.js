import Vue from 'vue';
import Vuetify from 'vuetify';
import { shallowMount } from '@vue/test-utils'
import { createRenderer } from 'vue-server-renderer';
import NavDrawerItems from '@/components/NavDrawer/Items/NavDrawerItems.vue'

Vue.use(Vuetify)
let wrapper

describe('NavDrawerItems.vue', () => {
  beforeEach(() => {
    wrapper = shallowMount(NavDrawerItems)
  })

  afterEach(() => {
    wrapper.destroy()
  })

  it('is a Vue instance', () => {
    expect(wrapper.isVueInstance()).toBeTruthy();
  })

  it('NavDrawerItems renders the same HTML', () => {
    const renderer = createRenderer();
    renderer.renderToString(wrapper.vm, (err, str) => {
    if (err) throw new Error(err);
      expect(str).toMatchSnapshot();
    });
  });
})
