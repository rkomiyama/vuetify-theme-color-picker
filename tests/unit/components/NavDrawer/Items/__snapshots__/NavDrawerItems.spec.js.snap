// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`NavDrawerItems.vue NavDrawerItems renders the same HTML 1`] = `
<v-list-stub tag="div" tile="true" dense="true" expand="true" nav="true">
  <SearchListGroup-stub></SearchListGroup-stub>
  <ThemeListGroup-stub></ThemeListGroup-stub>
</v-list-stub>
`;
